randovania/data/json_data/prime1.txt

```diff
@@ -3770,47 +3793,78 @@ Asset id: 1724639367
 ----------------
 Reactor Core
 Asset id: 3993592474
-> Dock to Reactor Access; Heals? False; Spawn Point
+> Dock to Reactor Access; Heals? False
   * Normal Door to Reactor Access/Dock to Reactor Core
-  > Dock to Main Ventilation Shaft Section A
-      Space Jump Boots
+  > Room Bottom
+      Trivial
 
-> Dock to Main Ventilation Shaft Section A; Heals? False
+> Dock to Main Ventilation Shaft Section A; Heals? False; Spawn Point
   * Normal Door to Main Ventilation Shaft Section A/Dock to Reactor Core
+  > Room Bottom
+      Trivial
+
+> Room Bottom; Heals? False
   > Dock to Reactor Access
-      Wave Beam and Thermal Visor and Space Jump Boots
+      All of the following:
+          After Reactor Core Conduits Activated
+          Space Jump Boots or Gravity Suit or Slope Jump (Advanced)
+  > Dock to Main Ventilation Shaft Section A
+      Space Jump Boots or Gravity Suit
+  > Event - Reactor Core Conduits Activated
+      All of the following:
+          Wave Beam
+          Thermal Visor or Invisible Objects (Beginner)
+          Space Jump Boots or Gravity Suit or Slope Jump (Advanced)
+
+> Event - Reactor Core Conduits Activated; Heals? False
+  * Event Reactor Core Conduits Activated
+  > Room Bottom
+      Trivial
 
 ----------------
 Reactor Access
 Asset id: 4215436672
-> Dock to Cargo Freight Lift to Deck Gamma; Heals? False; Spawn Point
+> Dock to Cargo Freight Lift to Deck Gamma; Heals? False
   * Normal Door to Cargo Freight Lift to Deck Gamma/Dock to Reactor Access
   > Dock to Reactor Core
       Trivial
-  > Dock to Savestation
-      Trivial
 
-> Dock to Reactor Core; Heals? False
+> Dock to Reactor Core; Heals? False; Spawn Point
   * Normal Door to Reactor Core/Dock to Reactor Access
   > Dock to Cargo Freight Lift to Deck Gamma
-      Wave Beam and Thermal Visor
+      After Reactor Access Conduits Activated
   > Dock to Savestation
       Trivial
+  > Event - Reactor Access Conduits Activated
+      All of the following:
+          Wave Beam
+          Thermal Visor or Invisible Objects (Beginner)
 
 > Dock to Savestation; Heals? False
   * Normal Door to Savestation/Dock to Reactor Access
   > Dock to Reactor Core
       Trivial
 
+> Event - Reactor Access Conduits Activated; Heals? False
+  * Event Reactor Access Conduits Activated
+  > Dock to Reactor Core
+      Trivial
+
 ----------------
 Cargo Freight Lift to Deck Gamma
 Asset id: 934522854
 > Dock to Reactor Access; Heals? False; Spawn Point
   * Normal Door to Reactor Access/Dock to Cargo Freight Lift to Deck Gamma
   > Dock to Deck Beta Transit Hall
-      Wave Beam and Thermal Visor and Space Jump Boots and Gravity Suit
+      Space Jump Boots and Gravity Suit
   > Pickup (Energy Tank)
-      Missile
+      After Cargo Freight Lift to Deck Gamma Gate Destroyed
+  > Event - Gate Destroyed
+      Missile or Charge Beam
```

Comment:
There really isn't a need for this event, since it would be a one way path anyways.